third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+mod experiments;
```

Comment:
Older packages with a similar file had
`#![forbid(unsafe_code)]`
here.  Did you want to have unsafe code for some reason?

Reply:
We don't do this anymore.